#! /bin/sh /usr/share/dpatch/dpatch-run
## 09_mythvideo_ubuntu_directories.dpatch by laga <>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Change some paths to match Debian/Ubuntu better

@DPATCH@

Index: mythvideo/globals.cpp
===================================================================
--- a/mythvideo/mythvideo/globals.cpp	(revision 13505)
+++ b/mythvideo/mythvideo/globals.cpp	(working copy)
@@ -28,5 +28,5 @@
 #ifdef Q_WS_MACX
 const QString DEFAULT_VIDEOSTARTUP_DIR = QDir::homeDirPath() + "/Movies";
 #else
-const QString DEFAULT_VIDEOSTARTUP_DIR = "/share/Movies/dvd";
+const QString DEFAULT_VIDEOSTARTUP_DIR = "/var/lib/mythtv/videos";
 #endif
